src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello, world!</title>
+  </head>
+  <body>
+    Hello, world!
```

Comment:
i'd rather do `<h1>Hello, world!</h1>`

Reply:
Thanks for your feedback ☺️ I've got some problems with deploy every time I run it so I'd like to discuss it at Q&A session 🙃